modules/ROOT/pages/managing-instances/migration-readiness.adoc

```diff
@@ -0,0 +1,102 @@
+= Migration Readiness Report
+:description: This section describes migration readiness report for upgrading from aura 4 to latest.
+:page-aliases: migration-readiness.adoc
+
+[NOTE]
+====
+The Migration Readiness Report is only available for Instances on Neo4j version 4.
+You can access the report for any version 4 instance via the link through the "Migration recommended" label on the Instance and Console pages.
+====
+
```

Comment:
```suggestion

[NOTE]
====
In the Migration Readiness Report we only highlight issues that need to be changed to make your code and queries compatible with AuraDB v5.
While you are working on this, we would also encourage you to extend this work to take advantage of the new v5 features.
====

```

Reply:
This sounds too much like marketing and it is not good UX to add one admonition right after another. Let me see if I can somehow convey this information in a different way.

Reply:
I've added a paragraph about this instead of in the admonition.